source/features/dim-bots.tsx

```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const commitSelector = [
-	/* Commits */
-	'.commit-author[href$="%5Bbot%5D"]',
-	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
-].join(',');
+const commitSelectors = [
+	'actions-user',
+	'bors',
+	'imgbot',
+	'octomerger',
+	'renovate-bot',
+	'rust-highfive',
+	'rust-lang',
```

Comment:
And I never get to resolve this one on my to-do list (it's a mistake, @rust-lang is a organization and won't be a bot)